### ISSUE TITLE
Problem running on Ubuntu 14.04 64bit

### ISSUE THREAD

**Issue by pdiffley** [uncomplicate/clojurecl#13]
I have followed the instructions on the getting started guide, but am having an error similar to that found in issue https://github.com/uncomplicate/clojurecl/issues/4. When I run

`(ns example
  (:use uncomplicate.clojurecl.core core info))`

I get the following error:

#error {
 :cause Error while loading native library "JOCL_2_0_0-linux-x86_64"
Operating system name: Linux
Architecture         : amd64
Architecture bit size: 64

I am running a laptop with an Intel Core i5-4200U cpu, and I have installed the Intel OpenCL SDK for Ubuntu (intel_sdk_for_opencl_2016_ubuntu_6.2.0.1760_x64.tgz).

Do I need to build the jocl library myself for this setup? Thank you!

**Comment by blueberry** [comment]
You should not have to build the library yourself. It should work out of the box if OpenCl is present. I've never tried Intel's SDK myself, so the first thing I'd check is whether it is properly installed and accessible from your system regardless of Clojure and JVM. Another thing is that Ubuntu seems a bit old (2 years is a lot of time in GPU computing).

Can you run clinfo from the terminal and what is its output?

Are you using leiningen and can you post your project.clj here?

**Comment by pdiffley** [comment]
Running clinfo I received the following output:

Number of platforms                               0

which I am assuming is the issue. I upgraded to Ubuntu 16.04, verified the Intel SDK was still installed, and received the same output from clinfo. I tried on a different computer running Ubuntu 16.04 with an AMD GPU. clinfo recognized a platform and device on this computer and the ClojureCL libraries loaded without issue.
